build.sbt

```diff
@@ -1,110 +1,153 @@
+import Base._
 import com.typesafe.sbt.pgp.PgpKeys.publishSigned
 import com.typesafe.sbt.SbtSite.SiteKeys._
 import com.typesafe.sbt.SbtGhPages.GhPagesKeys._
-import pl.project13.scala.sbt.SbtJmh._
 import sbtunidoc.Plugin.UnidocKeys._
 import ScoverageSbtPlugin._
 import ReleaseTransformations._
 
-lazy val scoverageSettings = Seq(
-  ScoverageKeys.coverageMinimum := 60,
-  ScoverageKeys.coverageFailOnMinimum := false,
-  ScoverageKeys.coverageHighlighting := scalaBinaryVersion.value != "2.10")
+
+/**
+ * These aliases serialise the build for the benefit of Travis-CI, also useful for pre-PR testing.
+ * If new projects are added to the build, these must be updated.
+ */
+addCommandAlias("buildJVM", ";coreJVM/compile;lawsJVM/compile;testsJVM/test")
+addCommandAlias("validateJVM", ";scalastyle;buildJVM")
+addCommandAlias("validateJS", ";coreJS/compile;lawsJS/compile;testsJS/test")
+addCommandAlias("validate", ";validateJS;validateJVM")
+
+
+/**
+ * Build settings
+ */
+val home = "https://github.com/non/alleycats"
+val repo = "git@github.com:non/alleycats.git"
+val api = "https://non.github.io/alleycats/api/"
+val license = ("MIT", url("http://opensource.org/licenses/MIT"))
+
+val catsVersion = "0.2.0"
+val disciplineVersion = "0.4"
+val kindProjectorVersion = "0.6.3"
+val machinistVersion = "0.4.1"
+val macroCompatVersion = "1.0.2"
+val paradiseVersion = "2.1.0-M5"
+val scalacheckVersion = "1.12.4"
+val scalatestVersion = "3.0.0-M7"
+val scalacVersion = "2.11.7"
+val simulacrumVersion= "0.4.0"
 
 lazy val buildSettings = Seq(
   organization := "org.spire-math",
   scalaVersion := "2.11.7",
-  crossScalaVersions := Seq("2.10.5", "2.11.7")
+  crossScalaVersions := Seq("2.10.6", scalacVersion)
```

Comment:
actually, it should be `scalaVersion := scalacVersion` in the line above, and yes, can then also do as you suggest